auth.php

```diff
@@ -457,6 +458,14 @@ function sync_users($do_updates=true) {
                     }
 
                     $id = user_create_user($user, false);
```

Comment:
If you have a really close look at the upstream changeset in https://github.com/moodle/moodle/commit/de91d620f2854ad9b1237c26df3ad110d89b9c44#diff-be62397967438defa4f9099b7176006e4b5603ae1da869930b74954d53a62cb8R958, you will see that you missed to comment out line 460 in your backport patch.